File: lat3_python/ram.py
class ram():

    __name = ""
    __price = ""

    def __init__(self, name = "", price = ""):
        self.__name = name
        self.__price = price

    def setName(self, name):
        self.__name = name

    def setPrice(self, price):
        self.__price = price

    def getName(self):
        return self.__name

    def getPrice(self):
        return self.__price

    def printRam(self):
        print("RAM Capacity     : ", str(self.getName()))
        print("RAM Proce        : ", str(self.getPrice()))

File: lat3_c++/Ram.cpp
#include<iostream>
#include<string>
using namespace std;

class Ram
{
private:
    string capacity;
    int price;

public:
    // konstruktor
    Ram() {}
    Ram(string capacity, int price)
    {
        this->capacity = capacity;
        this->price = price;
    }

    void setCapacity(string capacity)
    {
        this->capacity = capacity;
    }

    string getCapacity()
    {
        return this->capacity;
    }

    void setPrice(int price)
    {
        this->price = price;
    }

    int getPrice()
    {
        return this->price;
    }

    void outputRam()
    {
        cout << "Ram Capacity       : " << this->getCapacity() << endl;
        cout << "Ram Price          : " << this->getPrice() << endl;
    }

    // destructor
    ~Ram() {}
};
